### DISCUSSION TITLE
problems for building qt teamtalk

### DISCUSSION THREAD

**Discussion by HanaDadfar** [BearWare/TeamTalk5#2178] [Q&A]
### Description

currently, if we don't build teamtalk5.lib, the teamtalk5.vcxproj's build will fale, but if we don't have tolk, it builds with out any problem. Please make it so that the build doesn't go forward until the tolk is not found, i mean tolk.lib. I have problems that my teamtalk doesn't support tolk, so i can't add and test some things to the teamtalk that i whish. Thanks.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by HanaDadfar** [comment]
in the teamtalk5.pro or something.

**Comment by HanaDadfar** [comment]
I really hope this gets fixed, otherwise I won't be able to build qt teamtalk with tolk support, and until now i haven't receaved any help about this from anyone.

**Comment by bear101** [comment]
the .pro files are obsolete use CMake. The .pro files are only used for translation updates.

**Comment by HanaDadfar** [comment]
but when i looked at the codes inside it, i saw somthing about libs teamtalk5.lib. then, @bear101 please, kindely give me idea of how to fix this problems. You are the creator of teamtalk, if you don't help me, anyone else won't be able to help me. Thanks. Please, solv this problem, then my promice to you that i will contribute some needed things for teamtalk. Thanks.

**Comment by HanaDadfar** [comment]
Currently my built teamtalk files doesn't known the tolk as a dependency at all.
